### DISCUSSION TITLE
How to set Shorter Expiry / End session on Close

### DISCUSSION THREAD

**Discussion by daveteu** [nextauthjs/next-auth#7751] [Help]
Are sessions fixed on 30 days or are we able to set shorter expiry OR saving sessions in sessionStorage only?

I'm asking this because I wish to implement the "remember me for 30 days" feature, where sessions are close once browser closes (on public computers), unless opt in by users (safe computers).

Please point me to the right directions.

**Comment by jamaybyrone** [comment]
I'm also after something similar, I'd like the session cookie to end on browser close, which lead me to: https://github.com/nextauthjs/next-auth/discussions/3794. iron-session does have this feature https://github.com/vvo/iron-session#session-cookies

**Reply by dbstjrrb12** [reply]
@jamaybyrone  Is Possible to wrap NextAuth Option with withIronSessionApiRoute ?
